profile_library/epson/epson/ET-15000 Series/model.json

```diff
@@ -0,0 +1,17 @@
+{
+  "author": "Robert M. Albrecht <powercalc@romal.org>",
+  "calculation_strategy": "fixed",
+  "created_at": "2025-12-14T07:38:00",
+  "device_type": "printer",
+  "fixed_config": {
+    "states_power": {
+      "stopped": 0.54,
+      "idle": 7,
```

Comment:
What's the difference between these two states?

Reply:
> 1. Switched off (standby)
> 2. switched on but not printing
> 3. switched on and printing
> 

Looking at the screenshot, currently the epson is unavailable.
Did you actually disconnected it or is this some sort of sleep mode where wifi is turned off making the printer unavailable to HA.